### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by joserochh**
Support notice

### CODE REVIEW

**Review by faberga — CHANGES_REQUESTED** [review]
Hi @joserochh 

I understand your concern to inform the community that there is no official support for HEXL, but I'm not sure the tone of this notice is the right one.
Let me rephrase it, while communicating that the code is supplied "as is"

**Comment by faberga** [comment]
@joserochh  Section 7 of the license (See below) already states that HEXL is provided on "AS IS" basis. I will rephrase the README change to emphasize this point,

**7. Disclaimer of Warranty. Unless required by applicable law or
agreed to in writing, Licensor provides the Work (and each
Contributor provides its Contributions) on an "AS IS" BASIS,
WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
implied, including, without limitation, any warranties or conditions
of TITLE, NON-INFRINGEMENT, MERCHANTABILITY, or FITNESS FOR A
PARTICULAR PURPOSE. You are solely responsible for determining the
appropriateness of using or redistributing the Work and assume any
risks associated with Your exercise of permissions under this License.**

**Comment by joserochh** [comment]
@faberga Should we also rephrase Contributing section of the README to not encourage feedback or suggestion while there is no team to address those at the moment?

**Comment by faberga** [comment]
@joserochh, probably yes, but let me think about it a little more.
